experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -142,18 +143,16 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
               </td>
             </tr>
             <tr>
-              {experiment.targetingConfigSlug && (
-                <>
-                  <th>Advanced Targeting</th>
-                  <td data-testid="experiment-target">
-                    {displayConfigLabelOrNotSet(
-                      experiment.targetingConfigSlug,
-                      targetingConfigs,
-                    )}
-                  </td>
-                </>
-              )}
+              <th>First Run Release Date</th>
+              <td data-testid="experiment-release-date">
+                {experiment.proposedReleaseDate ? (
+                  experiment.proposedReleaseDate
+                ) : (
+                  <NotSet color="primary" />
+                )}
+              </td>
             </tr>
```

Comment:
if you will put colspan here, the missing line/border would come perfect- 
<img width="1724" alt="Screenshot 2023-06-13 at 1 50 59 PM" src="https://github.com/mozilla/experimenter/assets/25848231/fdcd92c7-9582-4d92-b4e7-bb1baba37edd">

Reply:
Done! 

<img width="1525" alt="image" src="https://github.com/mozilla/experimenter/assets/43795363/93d47e53-0c12-4a95-b967-8468b810b74a">